### PR TITLE
Align nvidia driver root envars

### PR DESCRIPTION
**Pull request by elezar**
Align the envvars used to specify the driver root with the NVIDIA Container Toolkit (NVIDIA_DRIVER_ROOT).

### CODE REVIEW

**Review by klueska — APPROVED** [review]

File: cmd/nvidia-mig-manager/main.go [hunk 1]
```diff
@@ -211,11 +211,10 @@ func main() {
 		},
 		&cli.StringFlag{
 			Name:        "driver-root",
-			Aliases:     []string{"t"},
-			Value:       DefaultDriverRoot,
-			Usage:       "Root path to the NVIDIA driver installation. Only used if --cdi-enabled is set.",
```
Comment (klueska) [inline cmd/nvidia-mig-manager/main.go:216]: Why do you drop the usage string?

File: cmd/nvidia-mig-manager/main.go [hunk 2]
```diff
@@ -211,11 +211,10 @@ func main() {
 		},
 		&cli.StringFlag{
 			Name:        "driver-root",
```
Comment (klueska) [inline cmd/nvidia-mig-manager/main.go:213]: If we consider the new value the "correct" value, should we change this to `nvidia-driver-root` and make the alias `driver-root`?

File: cmd/nvidia-mig-manager/main.go (hunk 1 above)
Comment (elezar) [reply cmd/nvidia-mig-manager/main.go:216]: Not intentional ... will readd.

File: cmd/nvidia-mig-manager/main.go (hunk 2 above)
Comment (elezar) [reply cmd/nvidia-mig-manager/main.go:213]: Yes. We should.

**Pull request merged**